SmartDeviceLink/SDLLifecycleConfiguration.h

```diff
@@ -139,7 +139,12 @@ typedef NS_OPTIONS(NSUInteger, SDLSecondaryTransports) {
 @property (copy, nonatomic, nullable) NSArray<SDLAppHMIType> *additionalAppTypes;
 
 /**
- *  The default language to use
+ *  The default hmi language to use
+ */
+@property (strong, nonatomic) SDLLanguage hmiLanguage;
```

Comment:
This was not an agreed upon change, please revert.

Reply:
OK. This has been reverted.